cli/tests/subdir/mod1.bundle.js

```diff
@@ -0,0 +1,178 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
```

Comment:
This file is the output of deno bundle command? I think it would be good to automatically generate it during the tests.

Reply:
:+1:

Reply:
The only problem is that we don't support a pre-req in our integration tests, and I can't be guaranteed execution order or dependency.  So to get one integration test to output it, and one integration test to consume it seems fragile, right?

Reply:
If you run and consume in a single test it wouldn't be fragile.

Reply:
Yes, but our integration tests don't support two invocations of Deno.

Reply:
You’d need to write a custom one. See for example the js_unit_tests https://github.com/denoland/deno/blob/master/cli/tests/integration_tests.rs#L41